evaluator/Dispatchers/abstract_dispatcher.py

```diff
@@ -75,15 +76,14 @@ def stop(self):
 
 
 # ****** Implementation side ******
-class DispatcherAbstractClient(AsynchronousServer):
```

Comment:
We can now remove the `AsynchronousServer` file in tool, can we ?

Reply:
+1